README.md

```diff
@@ -4,288 +4,7 @@ NUbots is a team in the University of Newcastle's robotics research group focuse
-     wget https://git.io/JeWaF -O ./arch_install.sh
-     chmod +x ./arch_install.sh
-     ```
-
-1. Execute the script and follow the instructions
-   - ```sh
-     ./arch_install.sh
-     ```
-
-The installation script will end by downloading a secondary script and providing you with a command that you must run
-
-```sh
-ROBOT_NUMBER=<N> arch-chroot /mnt ./arch-chroot_install.sh
-```
-
-Substitute `<N>` with the number of the robot that you are building. This will influence the IP address of the robot as well as the robots' hostname.
-
-Finally, once that script has finished, you must run one more command.
-
-```sh
-/mnt/arch-post_install.sh
-```
-
-This command will end by rebooting the robot. When this happens be sure to remove the USB installation drive from robot so that you may boot into the new system.
-
-The final two scripts can be found at [doc/ArchInstall/arch-chroot_install.sh](doc/ArchInstall/arch-chroot_install.sh) and [doc/ArchInstall/arch-post_install.sh](doc/ArchInstall/arch-post_install.sh)
-
-## Setting up WiFi manually
-
-To set up the WiFi interface you first need to know the name of the interface. To find this, run
-
-```sh
-ip link
-```
-
-and look for an interface starting with `wl`. On our robots it is usually `wlp58s0`.
-
-The rest of these instructions assume a network using WPA2.
-
-1. Make sure the `wpa_supplicant` configuration directory exists
-
-   - ```sh
-     mkdir /etc/wpa_supplicant
-     ```
-
-1. Setup the network configuration
-
-   - ```sh
-     wpa_passphrase <Network SSID> <Network Passphrase> > /etc/wpa_supplicant/wpa_supplicant-<WiFi Interface>.conf
-     ```
-   - Be sure to replace `<Network SSID>` with the SSID of the wireless network to connect to, `<Network Passphrase>` with the password for the network, and `<WiFi Interface>` with the name of the interface found earlier.
-
-1. Now start all of the necessary services
-
-   - ```sh
-     systemctl start dhcpcd.service
-     systemctl start wpa_supplicant.service
-     systemctl start wpa_supplicant@<Wifi Interface>.service
-     ```
-   - Be sure to replace `<WiFi Interface>` with the name of the interface found earlier.
-   - Wait a handful of seconds and run `ip addr` to check that the WiFi interface has an IP address
-
----
-
-<details>
-  <summary>Undocumented options</summary>
-
-## Shell access
-
-For some reason, if you really, really, _REALLY_ need to access a terminal inside of Docker you can run
-
-```sh
-./b shell
-```
-
----
-
-**NOTE**
-Any changes you make outside of the `/home/nubots/NUbots` or `/home/nubots/build` directories will not persist after exiting the Docker shell.
-
----
-
-</details>
+[NUbook](https://nubook.nubots.net/) is the handbook and high-level documentation for the NUbots team.
+It contains guides for [getting started with development](https://nubook.nubots.net/guides/main/getting-started) and [information about the team](https://nubook.nubots.net/team/introduction) for anyone interested in any aspect of it.
```

Comment:
```suggestion
It contains guides for [getting started with development](https://nubook.nubots.net/guides/main/getting-started) and [information about the team](https://nubook.nubots.net/team/introduction).
```
I didn't like the way it sounded. But feel free to dismiss.

Reply:
up to you @JosephusPaye

Reply:
Looks good, was trying to keep the general gist of your original @KipHamiltons.

Reply:
Appreciate it! I am easy though. I'll commit this suggestion.